content/ecosystem/bridges/discord/bridges.toml

```
@@ -12,16 +12,16 @@ featured = true
 privilege.platform = "Admin" # Free text
 privilege.matrix = "Homeserver Admin" # Any of Homeserver Admin, Room Admin, None
 [bridges.supports]
-dm = true
+dm = false
 channels = true
 formatted_text = true
 message_media = true
-replies = true
+replies = false # maybe?
```

```suggestion
replies = false
```
https://github.com/matrix-org/matrix-appservice-discord/issues/631#issuecomment-1236047430